examples/example-uvtm.c

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
We have an existing `example-libuv.c`, rename this to something like `example-libuv-timeout.c`?